.gitlab-ci.yml

```diff
@@ -78,7 +85,9 @@ check-wasm:
 build-std:
   stage:                           workspace
   <<:                              *docker-env
-  dependencies:
+  # dependencies:
+  #   - check-std
+  needs:
     - check-std
```

Comment:
What is the semantics of this? Can you shortly explain? And what is the difference to the former `dependencies`?

Reply:
It launches right after the mentioned job succeeds, doesn't wait for it's stage to be over.